funnel/templates/session_form.html.jinja2

```diff
@@ -1,7 +1,13 @@
-{% from "baseframe/forms.html.jinja2" import renderform_inner %}
-<form id="{{ formid }}" method="POST" action="javascript:void(0);" accept-charset="UTF-8" class="form-horizontal">
-    {{ renderform_inner(form, formid or None) }}
-</form>
-<script>
-    SHOW_SAVE_BUTTON = true;
-</script>
+{% from "baseframe/forms.html.jinja2" import renderform_inner, rendersubmit %}
+
+{% block content %}
+	<form id="{{ formid }}" method="POST" accept-charset="UTF-8" class="mui-form">
+	  {{ renderform_inner(form, formid or None) }}
+	</form>
+{% endblock %}
+
+{% block footerscripts %}
+	<script>
+	  SHOW_SAVE_BUTTON = true;
+	</script>
```

Comment:
If you have a script tag without content type like this, how does it work?

Reply:
from html5, missing `type` attribute indicates the script is JavaScript
But I will add the `type` attribute where ever it is missing.